### ISSUE TITLE
Specifying min_delay during setup gives an error due to the new gap junctions in NEST 2.10.0

### ISSUE THREAD

**Issue by jhnnsnk** [NeuralEnsemble/PyNN#400]
The support for simulations of gap junctions (connections without delays) is a new feature of NEST 2.10.0.
PyNN needs to adapt when calling `nest.SetDefaults()` for this synapse model if the keyword `min_delay` is specified during setup:

```python
import pyNN.nest as sim
sim.setup(min_delay=0.1)
```
results in the following error with PyNN 0.8.0:
```python
NESTError: BadProperty in SetDefaults_l_D: Setting status of prototype 'gap_junction': gap_junction connection has no delay
```

**Comment by apdavison** [comment]
I think this was already fixed in the master branch (see 42ad7b283eddac192de43b4300b5de3402efe478). Please could you try with the latest master and see if it works?

**Comment by jhnnsnk** [comment]
Yes, it works with the latest master. Sorry for the confusion.
